src/dodal/devices/turbo_slit.py

```diff
@@ -0,0 +1,20 @@
+from ophyd_async.core import Device
+from ophyd_async.epics.motion.motor import Motor
+
+
+class TurboSlit(Device):
+    """
+    This collection of motors coordinates time resolved XAS experiments.
+    It selects a beam out of the polychromatic fan.
+    These slits can be scanned continously or in step mode.
+    The relationship between the three motors is as follows:
+        - gap provides energy resolution
+        - xfine selects the energy
+        - arc - ???
```

Comment:
Should: Device description wants to be describing the device, information about how it is used for XAS experiments on i20-1 can go into the docstring of the device factory function in the beamline module.

Must: Fill in arc, ask i20-1 staff for description of device to make sure this is all accurate

Reply:
Tuesday 16.04 meeting with @tomtrafford  and beamline scientists should inform this

Reply:
comments added